python/cac_tripplanner/destinations/migrations/0043_auto_20180506_2103.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-05-07 01:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('destinations', '0042_userflag'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DestinationUserFlags',
+            fields=[
+            ],
+            options={
+                'verbose_name': 'Destination User Flag Summary',
+                'proxy': True,
+                'verbose_name_plural': 'Destination User Flags Summary',
+                'indexes': [],
+            },
+            bases=('destinations.destination',),
+        ),
+        migrations.CreateModel(
+            name='EventUserFlags',
+            fields=[
+            ],
+            options={
+                'verbose_name': 'Event User Flag Summary',
+                'proxy': True,
+                'verbose_name_plural': 'Event User Flags Summary',
+                'indexes': [],
+            },
+            bases=('destinations.destination',),
```

Comment:
Deleting and recreating this migration results in this same `destination` base, so it's not stale.